src/nodes/YAMLSeq.ts

```diff
@@ -116,6 +117,22 @@ export class YAMLSeq<T = unknown> extends Collection {
       onComment
     })
   }
+
+  static from(schema: Schema, obj: unknown, ctx: CreateNodeContext) {
+    const { replacer } = ctx
+    const seq = new YAMLSeq(schema)
```

Comment:
```suggestion
    const seq = new this(schema)
```

Presumably just an oversight? We'll want sequences to also be extendable.

Reply:
Good catch! Yes.